storage/kv/merkletreekv/storage.go

```diff
@@ -0,0 +1,12 @@
+package merkletreekv
+
+const (
+	EmptyNodeIdentifier    = 'E'
+	LeafNodeIdentifier     = 'L'
+	InteriorNodeIdentifier = 'I'
+	NodeKeyIdentifier      = 'N'
+	STRIdentifier          = 'S'
+	PoliciesIdentifier     = 'P'
+	EpochIdentifier        = 'O'
+	TreeNonceIdentifier    = 'T'
```

Comment:
Arent' those constants the same as in https://github.com/coniks-sys/coniks-go/blob/84fc324bc27fee50c0619a9538c0d91421b804d1/merkletree/storage.go ?